### PR TITLE
Implemented front-end to create a question#

### PR DESCRIPTION
**Pull request by jessiewon**
**What does this PR do?**
This PR has a slightly updated UI, with modifications to the profile buttons, as well as the front-end create a question functionality. Currently, it posts the question inputted and whether or not the question is anonymous to the database, but the user_id portion is not completed (it automatically sends user_id = 23 since user_id is a required field).

**How should this be manually tested?**
It's difficult to test since the database is local. However, I tested it with our current backend, which is why small changes are being made to backend and it is not complete.

**Relevant tickets:**
https://wizeline.atlassian.net/browse/WIP-189

### CODE REVIEW

File: src/components/List.js [hunk 1]
```diff
@@ -69,7 +53,7 @@ class List extends Component {
             </tr>
           </thead>
           <tbody>
-            {this.createList(filtered)}
+            {this.createList(this.state.questionData)}
```
Comment (codinronan) [inline src/components/List.js]: @jessiewon Can you make sure you've rebased from master properly? Mike implemented work that changed this to implement searching, which I think your PR has now lost.

@rominavarela

File: src/components/QuestionInput.js [hunk 2]
```diff
@@ -2,10 +2,12 @@ import React, { Component } from 'react';
 import axios from 'axios';
 import '../styles/QuestionInput.css';
 
+const qs = require('qs');
+
 class QuestionInput extends Component {
   constructor(props) {
     super(props);
-    this.state = { inputValue: '' };
+    this.state = { inputValue: '', isChecked: true };
```
Comment (codinronan) [inline src/components/QuestionInput.js]: Let's name this something more accurate, like `isAnonymousChecked`

File: src/components/QuestionInput.js [hunk 3]
```diff
@@ -14,14 +16,22 @@ class QuestionInput extends Component {
     this.setState({ inputValue: event.target.value });
   }
 
+  toggleChange = () => {
+    this.setState({
+      isChecked: !this.state.isChecked,
```
Comment (codinronan) [inline src/components/QuestionInput.js]: Better to read the value from the checkbox, than to simply toggle the flag. There are some gotchas with these things..

**Comment by codinronan** [comment]
@wizevu Can you check this? I think some of your searching changes are not here.

File: src/components/List.js [hunk 4]
```diff
@@ -69,7 +53,7 @@ class List extends Component {
...
           </tbody>
         </table>
       </div>
```
Comment (wizevu) [inline src/components/List.js]: Please discard these changes here to keep the search functionality!

**Comment by codinronan** [comment]
@jessiewon LGTM! Go ahead and merge.

FYI let's use more descriptive messages than "	Made updates for PR". Eventually, this won't mean much when looking at the commit history, whereas "Fix search functionality and variable naming" (for example) would be more meaningful.

Anyway, looks great, thanks!

**Pull request merged**